library/tasks/RSEM.wdl

```diff
@@ -29,7 +29,7 @@ task RSEMExpression {
   command {
     set -e
   
-    tar -xvf ${rsem_genome}
+    tar --no-same-owner -xvf ${rsem_genome}
```

Comment:
`rsem` and `hisat2` indexing files are `tar.gz`. is it okay to just use `tar -xvf` to extract `.tar.gz` file?

Reply:
Good point here! If they are gzip files, `-z` will be necessary here I guess

Reply:
You actually don't need the `-z` for gnu tar. See here: https://www.gnu.org/software/tar/manual/tar.html#SEC135

"Reading compressed archive is even simpler: you don't need to specify any additional options as GNU tar recognizes its format automatically."

Also, I think the PR tests would have failed.

Reply:
I see, thanks!

Reply:
good to know this, thanks!